services/minespace-web/src/components/pages/Incidents/IncidentPage.js

```diff
@@ -325,7 +325,7 @@ const StepForms = (
           okText="Yes"
           cancelText="No"
         >
-          {!props.incident?.status_code && (
+          {props.incident.status_code === "DFT" && (
```

Comment:
not a status to a specific status code, again not sure I understand this change.